### PR TITLE
Add husarion_ugv_msgs

### PR DESCRIPTION
**Pull request by rafal-gorecki**
### Description

- add husarion_ugv_msgs
- repleace all occurance panther_msgs -> husarion_ugv_msgs

### Requirements

- [x] Code style guidelines followed
- [x] Documentation updated

### Tests 🧪

- [x] Robot (build in clean workspace test pass)
- [ ] Container
- [x] Simulation

## Release Notes

- **New Features**
	- Introduced new message types in the `husarion_ugv_msgs` package, including `ChargingStatus`, `DriverState`, `FaultFlag`, `IOState`, `LEDAnimation`, `RobotDriverState`, and `SystemStatus`.
	- Updated service definitions for `SetLEDAnimation`, `SetLEDBrightness`, and `SetLEDImageAnimation` to enhance functionality.

- **Documentation**
	- Updated documentation to reflect changes in message types and service calls across various packages.

- **Chores**
	- Removed deprecated dependencies on `panther_msgs` and replaced them with `husarion_ugv_msgs` across multiple packages.
	- Modified URLs in package metadata to remove trailing slashes.

### CODE REVIEW

File: ROS_API.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Below is information about the physical robot API. For the simulation, topics an
...
-| ✅   | ❌   | `battery/charging_status`     | Battery charging status value.<br/> [*panther_msgs/ChargingStatus*](https://github.com/husarion/panther_msgs)                                                                                                                                                                                                                                                             |
+| ✅   | ❌   | `battery/charging_status`     | Battery charging status value.<br/> [*husarion_ugv_msgs/ChargingStatus*](https://github.com/husarion/husarion_ugv_msgs)                                                                                                                                                                                                                                                             |
```
Comment (KmakD) [inline ROS_API.md]: Link should point to `panther_ros` repo and `husarion_ugv_msgs` package. Applies to all links

File: husarion_ugv_lights/test/integration/husarion_ugv_lights.test.py [hunk 2]
```diff
@@ -28,11 +28,12 @@
 from launch_ros.actions import Node
 from launch_ros.substitutions import FindPackageShare
 from launch_testing_ros import WaitForTopics
-from panther_msgs.msg import LEDAnimation
-from panther_msgs.srv import SetLEDAnimation
 from sensor_msgs.msg import Image
 from std_srvs.srv import SetBool
```
Comment (KmakD) [inline husarion_ugv_lights/test/integration/husarion_ugv_lights.test.py:32]: could also add line between launch imports and standard msgs

**Review by KmakD — CHANGES_REQUESTED** [review]
Forgot to mark this as request changes

File: husarion_ugv_lights/test/integration/husarion_ugv_lights.test.py (hunk 2 above)
Comment (rafal-gorecki) [reply husarion_ugv_lights/test/integration/husarion_ugv_lights.test.py:32]: pre-commit reformat it that way

Comment (rafal-gorecki) [reply husarion_ugv_lights/test/integration/husarion_ugv_lights.test.py:32]: by default isor order look like this:
```python
from __future__ import annotations  # FUTURE

import os  # STDLIB
import sys  # STDLIB

import numpy as np  # THIRDPARTY
import rospy  # THIRDPARTY

from my_project.module import MyClass  # FIRSTPARTY

from .local_module import MyLocalClass  # LOCALFOLDER
```
IMO there is no need to complicate that.

File: ROS_API.md [hunk 3]
```diff
@@ -68,7 +68,7 @@ Below is information about the physical robot API. For the simulation, topics an
...
-| ✅   | ❌   | `battery/charging_status`     | Battery charging status value.<br/> [*panther_msgs/ChargingStatus*](https://github.com/husarion/panther_msgs)                                                                                                                                                                                                                                                             |
+| ✅   | ❌   | `battery/charging_status`     | Battery charging status value.<br/> [*husarion_ugv_msgs/ChargingStatus*](husarion_ugv_msgs)                                                                                                                                                                                                                                                             |
```
Comment (KmakD) [inline ROS_API.md]: should we also change it so it points to the msg file instead of just the package?

**Review by KmakD — APPROVED** [review]
lgtm

**Pull request merged**